recipes-wigwag/maestro/maestro/maestro.service

```diff
@@ -5,6 +5,7 @@ Descritpion=Maestro: Network, Config, DeviceJS manager
 Restart=always
 RestartSec=5s
 Environment="LD_LIBRARY_PATH=/wigwag/system/lib"
+ExecStartPre=mkdir -p /userdata/etc/
```

Comment:
Why would this cause a segfault?

Reply:
Maestro creates database in /userdata/etc/maestroConfig.db and if the directory is not already created, it crashes.

Reply:
That's weird. I feel like if an application expects a file to be somewhere, it should check to make sure it exists first before attempting to do something.

Thanks for the context.